Develop2

apologet-insane opened the pull request.

apologet-insane committed:
проверка глючащего гитхаба

apologet-insane committed:
Тесты готовы. Отчёт добавлен в пуш.

apologet-insane committed:
1. Добавлены классы с методами
2. Форматирование исправлено

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
-    @Step ("Тело для изменения имени")
+    @Step("Тело для изменения имени")
     public String testPositiveUserName() {
         String registerRequestBody = "{\"name\":\"" + userName + userName + "\"}";
```
убери лишний userName

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
+    @Step("Тело для изменения email")
     public String testPositiveUserMail() {
+
         String registerRequestBody = "{\"email\":\"" + userName + userMail + "\"}";
```
убери userName

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
+    @Step("Тело для изменения пароля")
     public String testPositiveUserPassword() {
+
         String registerRequestBody = "{\"password\":\"" + userName + userPassword + "\"}";
```
убери userName

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
-   @Step
-    public void tearDown(){
+    @Step("Если поместить этот метод в After, все тесты падают")
+    public void tearDown() {
```
Если указать аннотацию @step этот метод выполняться не будет. Тесты падают по другим причинам.

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
+                .extract()
+                .path("success");
 
         assertThat(String.valueOf(success), true);
```
Неправильно указаны параметры в методе. Данная проверка е будет фейлиться. Исправь на:
assertThat(success, equalTo(true))

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
+                .extract()
+                .path("success");
 
         assertThat(String.valueOf(success), true);
```
Неправильно указаны параметры в методе. Данная проверка е будет фейлиться. Исправь на:
assertThat(success, equalTo(true))

nikolay-977 commented on src/test/java/PatchUserTest.java:
```diff
@@ -17,163 +18,175 @@
+                .path("success");
 
-      assertThat(String.valueOf(success), true);
+        assertThat(String.valueOf(success), true);
```
Неправильно указаны параметры в методе. Данная проверка е будет фейлиться. Исправь на:
assertThat(success, equalTo(true))

nikolay-977 commented on src/test/java/CreateUserTest.java:
```diff
@@ -118,104 +99,79 @@ public void deleteTestUser2(){
 
-        String registerRequestBody =  "{\"name\":\"" + userName + "\","
+        String registerRequestBody = "{\"name\":\"" + userName + "\","
                 + "\"passwoer\":\"" + userPassword + "\"}";
```
исправь опечатку в passwoer

apologet-insane committed:
1. Всё, что должно быть в @After теперь там
2. Исправил всё, что указано исправить
3. Все шаги в классах в main обозначены.
